polytope/polytope.py

```diff
@@ -772,21 +775,22 @@ def is_convex(reg, abs_tol=ABS_TOL):
     outer = envelope(reg)
     if is_empty(outer):
         # Probably because input polytopes were so small and ugly..
-        return False,None
+        return False, None
 
-    Pl,Pu = reg.bounding_box
-    Ol,Ou = outer.bounding_box
+    Pl, Pu = reg.bounding_box
+    Ol, Ou = outer.bounding_box
 
-    bboxP = np.hstack([Pl,Pu])
-    bboxO = np.hstack([Ol,Ou])
+    bboxP = np.hstack([Pl, Pu])
+    bboxO = np.hstack([Ol, Ou])
 
-    if sum(abs(bboxP[:,0] - bboxO[:,0]) > abs_tol) > 0 or \
-    sum(abs(bboxP[:,1] - bboxO[:,1]) > abs_tol) > 0:
-        return False,None
+    if sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or \
```

Comment:
I wasn't really targeting the removal of `\` at line continuations. I don't think the PEP8 extension in my editor marks them as incorrect, but I will remove/avoid them in the future.

I agree that it is more understandable if the line leads with the operator `or`. Is it possible to increase the indent of the `return` statement without the interpreter throwing an error?